src/olympia/api/tests/test_authentication.py

```diff
@@ -134,8 +135,9 @@ def test_cant_refresh_token(self):
         payload['orig_iat'] = timegm(payload['iat'].utctimetuple())
         token = self.encode_token_payload(payload, api_key.secret)
 
-        refresh_token_url = reverse('frontend-token-refresh')
-        response = self.client.post(refresh_token_url, data={'token': token})
+        request = self.factory.post('/lol-refresh', {'token': token})
+        response = refresh_jwt_token(request)
+        response.render()
         assert response.status_code == 400
         data = json.loads(response.content)
         assert data == {'non_field_errors': ['Error decoding signature.']}
```

Comment:
This assertion is  definitely weird though. Seems odd that it's returning that error.